api/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseInternalApi.java

```diff
@@ -80,4 +80,7 @@ public SubscriptionBaseBundle createBundleForAccount(UUID accountId, String bund
 
     public void updateExternalKey(UUID bundleId, String newExternalKey, InternalCallContext context);
 
+    public Iterable<DateTime> getFutureNotificationsForAccount(InternalCallContext context);
+
+    public Map<UUID, DateTime> getNextFutureEventForSubscriptions(final SubscriptionBaseTransitionType eventType, final InternalCallContext internalCallContext);
 }
```

Comment:
Sorry, our internal forks got out of sync.  Here's a view of only the Postgres changes - https://github.com/killbill/killbill/compare/master...marksimu:postgresql-queries